airflow_kubernetes_job_operator/kubernetes_job_operator.py

```diff
@@ -69,6 +76,12 @@ def __init__(
             validate_body_on_init {bool} -- If true, validates the yaml in the constructor,
                 setting this to True, will slow dag creation.
                 (default: {from env/airflow config: AIRFLOW__KUBE_JOB_OPERATOR__validate_body_on_init or False})
+            enable_jinja {bool} -- If true, the following fields will be parsed as jinja2,
+                        command, arguments, image, envs, body, namespace, config_file, cluster_context
```

Comment:
I don't think enabling jinja on anything but the body is useful. On the other hand, it would be useful to specify template parameters in this constructor, as a dict for example.

Reply:
You can in general specify template parameters, in general, for any airflow Task. 
```python
task = create_my_task()
task.template_fields=[*task.template_fields,"new field"]
```

This is the airflow way... so I think we should stick with it. I can add an input argument there... But the other default fields I created should not have jinja in them.

Reply:
You're right, I was not familiar with this functionality of Airflow. Works for me then!